news/2017-08-30-qat-gazette-august-2017.md

```diff
@@ -0,0 +1,41 @@
+# QAT Gazette - August 2017
+
+Hello!
+
+To those of you who have been following this blog at its’ [previous home](http://osuqat.tumblr.com/), welcome back! We’ve shuffled through a bit of a lean period on account of the summer holidays (or in my case, even more work than usual), but I am happy to bring you **the return of The QAT Gazette!** Feel free to skip down a little if you’re already familiar with our humble blog and want to get straight into the news, but if this is your first time and you’re wondering what all the fuss is about, read on!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
```

Comment:
banner should be exactly after the opener paragraph, should also be 750px width minimum

Reply:
Removed space, banner is currently 1000x300 so this should be ok unless you meant 750px maximum in which case I'll shrink it some more